website/models.py

```diff
@@ -638,6 +638,28 @@ def __str__(self):
         return f"Q: {self.question} | A: {self.answer} at {self.timestamp}"
 
 
+class Suggestion(models.Model):
+    user = models.CharField(max_length=100, default="Add user")
+    title = models.CharField(max_length=200)
+    description = models.TextField(max_length=1000, null=True, blank=True)
+    up_vote = models.IntegerField(null=True, blank=True, default=0)
+    down_vote = models.IntegerField(null=True, blank=True, default=0)
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
I'm not sure you need it. If you want uuid add it with a different name

Reply:
Isnt usig uuid a better option to create ids?

Reply:
It depends. Both have their pros and cons.